src/handlers/assign/action.ts

```diff
@@ -72,13 +73,19 @@ export const closePullRequestForAnIssue = async (): Promise<void> => {
   const payload = context.payload as Payload;
   if (!payload.issue?.number) return;
 
-  const prs = await getOpenedPullRequestsForAnIssue(payload.issue.number, "");
+  const prs = await gitIssueParser({
+    owner: payload.repository.owner.login,
+    repo: payload.repository.name,
+    issue_number: payload.issue.number,
+  });
+
```

Comment:
This requires organization & repo name check and so on. 

https://github.com/ubiquity/ubiquibot/pull/657

It is being updated as gitLinkedPrParser and gitLinkedIssueParser. You can wait till it is finished.

Reply:
But `gitLinkedPrParser` will only return the latest merged PR, this feature should close all PRs related to the issue

Reply:
> This requires organization & repo name check and so on. 
> 
> https://github.com/ubiquity/ubiquibot/pull/657
> 
> It is being updated as gitLinkedPrParser and gitLinkedIssueParser. You can wait till it is finished.

I don't think that'd work for this, this gets multiple linked PR. I think we should move on with this

Reply:
You can just refer them(gitLinkedPrParser, and issueParser) and add some features to gitIssueParser - maybe org & repo name check will be required.

Reply:
> You can just refer them(gitLinkedPrParser, and issueParser) and add some features to gitIssueParser - maybe org & repo name check will be required.

Done